.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     },
     "rules": {
         "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-        "prettier/prettier": ["error", { "singleQuote": true }, {"trailingComma" :true}]
+        "comma-dangle": ["error", "never"],
+        "prettier/prettier": ["error", { "singleQuote": true }]
```

Comment:
Is prettier going to place nice with airbnb styles? For some reason I thought that clashed, hopefully I'm just wrong :)